scripts/curl_install_pypi/install

```diff
@@ -24,13 +24,19 @@ then
   echo "$INSTALL_SCRIPT_SHA256  $install_script" | shasum -a 256 -c - || exit
 fi
 
-if ! command -v python >/dev/null 2>&1
+python_cmd=python3
```

Comment:
Is it matter to identify python or python3? Is possible to add shabang `#!/usr/bin/env python` instead?

Reply:
What `#!/usr/bin/env python` uses depends on user's setting of `python`. If `python` points to Python 2, then `scripts/curl_install_pypi/install.py` will be executed with Python 2. This script now prefers `python3`.